src/sc/sidechaintypes.h

```diff
@@ -483,6 +483,26 @@ typedef struct sScFeeData_tag
 
 } ScFeeData;
 
+struct ScFeeData_v2 : public ScFeeData {
+    int submissionHeight;
+    ScFeeData_v2() : ScFeeData(), submissionHeight(-1) {};
+    ScFeeData_v2(CAmount f, CAmount m, int submHeight) : ScFeeData(f, m), submissionHeight(submHeight) {};
+
+    ADD_SERIALIZE_METHODS;
+
+    template <typename Stream, typename Operation>
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
+        READWRITE(forwardTxScFee);
+        READWRITE(mbtrTxScFee);
```

Comment:
Just a small tip here, feel free to ignore. Have you considered something like the following?
```suggestion
        ScFeeData::SerializationOp(s, ser_action, nType, nVersion);
```
To me this also seems more in line with `ScFeeData_v2`'s constructors.